packages/fastify-renderer/src/node/renderers/Renderer.ts

```diff
@@ -9,6 +9,13 @@ export interface RenderOptions {
   base: string
 }
 
+export type PartialRenderOptions =
```

Comment:
Could you use `Partial<RenderOptions>` here (or in the place where this type is used)?

Reply:
My understanding of `Partial` is that it makes all properties optional and doesn't allow specifying dependencies between the fields (i.e. if the `layout` property is present then the `base` property is required).